manager/integration/tests/test_node.py

```diff
@@ -350,6 +350,27 @@ def test_replica_scheduler_no_disks(client):  # NOQA
     common.wait_for_volume_delete(client, vol_name)
 
 
+@pytest.mark.node  # NOQA
+def test_replica_zone_anti_affinity(client):  # NOQA
+    """
+    Test replica scheduler with zone anti-affinity
+
+    1. Set zone anti-affinity to hard, and replica count to 2.
```

Comment:
- we don't need to set default replica count to `2`, instead, we can create a volume with `2` replicas in step `3`

Reply:
I will move this to step 3, since we have 3 nodes we will need 3 replicas.